### PR TITLE
docs(components): [table-v2] unified display style of parameters

### PR DESCRIPTION
**Pull request by warmthsea**
#### Before

image

#### After

image

### CODE REVIEW

File: docs/en-US/component/table-v2.md [hunk 1]
```diff
@@ -295,70 +295,70 @@ table-v2/manual-scroll
...
+| ------------------------- | -------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------ | --------- |
+| cache                     | Number of rows rendered in advance to boost the performance                                                                | `number`                                               | 2         |
+| estimated-row-height      | The estimated row height for rendering dynamic height rows                                                                 | `number`                                               | —         |
+| header-class              | Customized class name passed to header wrapper                                                                             | `string` / Function<[HeaderClassGetter](#typings)>     | —         |
+| header-props              | Customized props name passed to header component                                                                           | `object` / Function< [HeaderPropsGetter](#typings)>    | —         |
```
Comment (btea) [inline docs/en-US/component/table-v2.md:303]:
```suggestion
| header-props              | Customized props name passed to header component                                                                           | `object` / Function<[HeaderPropsGetter](#typings)>    | —         |
```

**Review by btea — COMMENTED** [review]
Is there a better way to write this function type? It seems a bit strange at the moment.

QQ_1732025244044

**Comment by warmthsea** [comment]
> Is there a better way to write this function type? It seems a bit strange at the moment.

~~Function<[HeaderClassGetter]>~~ ——>  HeaderClassGetter
or like this（but bit long）
image

**Comment by btea** [comment]
It looks perfect, and it doesn’t matter if it’s a little longer when placed in a pop-up window. 👍

**Comment by warmthsea** [comment]
> It looks perfect, and it doesn’t matter if it’s a little longer when placed in a pop-up window. 👍


`TableV2` many types is  long, I think it's better not to change to  pop-up.

image

<hr />

image

**Review by btea — APPROVED** [review]
The current modification works fine for me.

**Pull request merged**